connectors/planetradio.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#app';
+
+Connector.artistSelector = '.text-right > span:nth-child(2)';
+
+Connector.trackSelector = '.text-right > span:nth-child(3)';
+
+Connector.trackArtImageSelector = 'div.now-playing-block:nth-child(3) > div:nth-child(1)';
```

Comment:
I suggest to use more straigtforward CSS selectors:
* `.text-right .title` for artist;
* `.text-right .track` for track;
* `.now-playing-block.right .image` for track art.

I also suggest to use `.now-playing-wrapper` as player selector.